tests/unit/ruby/try/16_config_emailer_try.rb

```diff
@@ -43,7 +43,7 @@
 
 ## Emailer values can be set via environment variables
 ENV['EMAILER_MODE'] = 'sendmail'
-ENV['FROM'] = 'test@example.com'
+ENV['FROM_EMAIL'] = 'test@example.com'
```

Comment:
Unless this was causing issues, let's leave the tests as is. 

```suggestion
ENV['FROM'] = 'test@example.com'
```